### ISSUE TITLE
Allow user-specified tile sources

### ISSUE THREAD

**Issue by GoogleCodeExporter** [MarcusWolschon/osmeditor4android#119]
```
Allow the user to enter extra background tile sources. The concept as 
implemented by Potlatch2 seems appropriate.

Need to consider how to manage the tile cache without important information 
such as image file types (image filename extension), min/max zoom levels, and 
how to name the cache folders in a systematic way.

We would most likely need to change the way we currently manage the tile cache 
and base folder names around the source URL. This would mean issue 44 becomes a 
WONTFIX - it would become impossible to share a tile cache with other 
applications.
```

Original issue reported on code.google.com by `andrew.g...@gmail.com` on 9 Sep 2012 at 4:11
- Blocked on: #213

**Comment by DFyson** [comment]
What's the URL of the imagery index file on github? Looking for the format of imagery.json

**Comment by simonpoole** [comment]
https://github.com/simonpoole/osm-layer-index
or
https://github.com/osmlab/editor-layer-index
